README.md

```diff
@@ -146,9 +122,16 @@ Before any packages can be used, the registry needs to be downloaded. This is do
 different Mason commands (e.g. `:MasonInstall`), but can also be done manually by using the `:MasonUpdate` command.
 
 If you're utilizing Mason's Lua APIs to access packages, it's recommended to use the
-[`:h mason-registry.refresh()`][help-mason-registry-refresh] and/or [`:h mason-registry.update()`][help-mason-registry-update]
+[`:h mason-registry.refresh()`][help-mason-registry-refresh] or [`:h mason-registry.update()`][help-mason-registry-update]
 functions to ensure you have the latest package information before retrieving packages.
 
+## Screenshots
```

Comment:
should be same heading-level as the rest, since this is not a subsection of "Registries"